tests/autoyast/installation.pm

```diff
@@ -52,18 +52,6 @@ sub save_and_upload_stage_logs {
 }
 
 sub upload_autoyast_profile {
-    my ($self) = @_;
-    select_console 'install-shell';
-    # the network may be down with keep_install_network=false
-    # use static ip in that case if not on s390x
-    if (!check_var("BACKEND", "s390x")) {
-        type_string " if ! ping -c 1 10.0.2.2 ; then
-            ip addr add 10.0.2.200/24 dev eth0
-            ip link set eth0 up
-            route add default gw 10.0.2.2
-        fi
-        ";
-    }
```

Comment:
could you explain in more detail why we don't need this section? It's not mentioned in the commit message so one of this changes within a commit message that promises a different thing.

Reply:
We already do network recovery in lib/y2logstep, so the only part which is different to interactive installation is collection of the profile.

Reply:
@okurz, I stated this in the PR message.

> Also, it removes redundant network reconnect, as it is completed on the
> 269th line of y2logsstep.pm ( if (can_upload_logs() ).